features/admin.applications.v1/pages/application-edit.tsx

```diff
@@ -57,13 +67,17 @@ import {
     ApplicationAccessTypes,
     ApplicationInterface,
     ApplicationTemplateListItemInterface,
+    OIDCDataInterface,
     State,
     SupportedAuthProtocolTypes,
     idpInfoTypeInterface
 } from "../models";
 import { ApplicationManagementUtils } from "../utils/application-management-utils";
 import { ApplicationTemplateManagementUtils } from "../utils/application-template-management-utils";
 import "./application-edit.scss";
+import { Typography } from "@mui/material";
```

Comment:
do not directly import from material, use `https://wso2.github.io/oxygen-ui/react/?path=/docs/data-display-typography--overview` instead.

Reply:
Fixed with 025df58bd7452704342e396767d22883776319ca